www/source/partials/docs/_bp-kubernetes.html.md.erb

```diff
@@ -3,7 +3,15 @@
 
 ## Habitat Kubernetes Operator
 
-The [Habitat Kubernetes Operator](https://github.com/kinvolk/habitat-operator) is on-going work to create an operator that leverages Kubernetes API services to create a native and robust integration between the two technologies. Follow along on [github](https://github.com/kinvolk/habitat-operator), and join us in our #kubernetes channel in the [Habitat Slack Channel](https://slack.habitat.sh) for more information.
+The [Habitat Kubernetes Operator](https://github.com/kinvolk/habitat-operator) is on-going work to create an operator that leverages Kubernetes API services to create a native and robust integration between the two technologies. Follow along on [github](https://github.com/kinvolk/habitat-operator), and join us in our [#kubernetes](https://habitat-sh.slack.com/messages/kubernetes/) channel in the [Habitat Slack](https://slack.habitat.sh) for more information.
```

Comment:
Think a bit more can be said here no? Like the architecture and about how it works. Link to the blog post about it. etc.

Reply:
I can link to the blog post, but I'm not sure that an architectural description is what the volks at Chef want here.

/cc @tashimi